packages/stryker-karma-runner/test/unit/KarmaTestRunnerSpec.ts

```diff
@@ -206,12 +195,6 @@ describe('KarmaTestRunner', () => {
       expect(TestHooksMiddleware.instance.currentTestHooks).eq('foobar');
     });
 
-    it('should update port when karma chooses a different port', async () => {
-      reporterMock.emit('server_start', 1984);
-      await sut.run({});
-      expect(karmaRunStub).calledWith({ port: 1984 });
```

Comment:
This is also still a valid test. The stryker-karma-runner remains interested in the port that karma chooses to claim.